pkg/elfparser/elf.go

```diff
@@ -247,9 +247,9 @@ func (e *elfLoader) loadProg(loadedProgData map[string]ebpf_progs.CreateEBPFProg
 
 	for _, pgmInput := range loadedProgData {
 		bpfData := BpfData{}
-		progFD, _ := e.bpfProgApi.LoadProg(pgmInput)
+		progFD, errno := e.bpfProgApi.LoadProg(pgmInput)
```

Comment:
Nit:- why not keep name of variable `err` instead of `errno`.

Reply:
Kept it as errno as its a number instead of an error message